analyses/fusion-summary/01-fusion-summary.Rmd

```diff
@@ -67,10 +91,14 @@ prepareOutput <- function(fuseDF, bioid) {
 
 ```{r}
 dataDir <- file.path("..", "..", "data")
+fusDir <- file.path("..", "..", "analyses", "fusion_filtering", "results")
+annotDir <- file.path("..", "..", "analyses", "fusion_filtering", "references")
 #' The putative oncogenic fusion file is what we'll use to check for the 
+#' annotation file
+annot <- read.delim2(file.path(annotDir, "genelistreference.txt"), sep = "\t", header = T)
 #' presence or absence of the fusions.
 putativeOncogenicDF <- 
-  read_tsv(file.path(dataDir, "pbta-fusion-putative-oncogenic.tsv"))
+  read_tsv(file.path(fusDir, "pbta-fusion-putative-oncogenic.tsv"))
```

Comment:
@jaclyn-taroni - some heads up:

I used `read.delim2()` here because `read_tsv()` gave me an error in that the first column was rownames and the last column was missing. Maybe you will know a workaround if you don't like read.delim2 :)

The putative oncogenic path will need to be changed with v18 update once we have the required columns, but for now, I pulled from the latest analysis.

Reply:
@cansavvy is going to provide a review, so maybe she will have some ideas!

Reply:
Where does `genelistreference.txt` come from? [The README says what it contains](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering#inputs-used-as-reference) but its unclear to me where its from -- there's a lot of references listed which is great but not info about how this file came to be. Anyway, `genelistreference.txt`  has numbering saved as a row name type of column which is why this is giving an error (it doesn't have a column name). It would be best if we just got rid of that numbering column since its not doing anything anyway. 

<img width="491" alt="Screen Shot 2020-11-06 at 9 21 45 AM" src="https://user-images.githubusercontent.com/23458084/98376682-d8dfa380-2011-11eb-8202-8c031c6736d6.png">

This might need to be its own issue (probably not something we should address in this PR).

Reply:
If we cannot or should not change the file, and we want to use readr::read_tsv(), then we can get around this by providing col_names and telling readr to skip whatever column name we give to the number column, but the better solution if possible is just to drop this column that doesn't give any info.

Reply:
The file was created from different sources and formatted [code](https://gist.github.com/kgaonkar6/02b3fbcfeeddfa282a1cdf4803704794) I can probably re-do that to save with write_tsv instead of write.table which I believed added the rownames (default row.names==TRUE)

Reply:
@kgaonkar6 you had cleaned up that file in [annofuse](https://github.com/d3b-center/annoFuse/issues/62) - will you add an issue and quick PR here?

Reply:
ok will do

Reply:
Sounds great! :+1:

Reply:
#836 submitted